sockets/challonge.js

```diff
@@ -18,9 +20,15 @@ module.exports = function(io) {
         socket.emit('update challonge', challongeData);
         connectedSockets++;
 
+        // if we go from 0 to 1 socket, start polling again
+        pollChallonge();
+
         socket.on('disconnect', function() {
             console.log('Challonge disconnected');
             connectedSockets--;
+            if (connectedSockets === 0) {
```

Comment:
What are the odds of `connectedSockets` not being safely write-locked and falling below 0? I'm imagining it should be fine, but that would break this.